CONTRIBUTING.md

```diff
@@ -20,12 +20,14 @@ To learn about ways to participate and interact with community members, see [Onl
 
 All contributors agree to respect and follow the Mina Protocol [Code of Conduct](https://github.com/MinaProtocol/mina/blob/develop/CODE_OF_CONDUCT.md).
 
-## Docs Toolset 
+## Docs Toolset
 
 The [Mina Docs](https://docs.minaprotocol.com/) website is published with the Docusaurus static site generator. The docs source files are written in Markdown `.mdx` files that support Markdown with embedded components through the JSX syntax extension to JavaScript. Docusaurus has built-in support for MDX that renders the JSX in Markdown files as React components. 
 
 Even if you're unfamiliar with Markdown, you can still contribute to the docs in a helpful and meaningful way. Use [Markdown Basic Syntax](https://www.markdownguide.org/basic-syntax) and look at existing docs for models to follow. You can start contributing by submitting fixes for typos, spelling, and broken links. 
 
+You can even build the docs locally. See the [README](readme) file. 
```

Comment:
Hmm, I wonder about this one. I appreciate and agree with the sentiment but that 'even' is a bit jarring to me. It implies that this is novel, whereas I think it's reasonable to expect that docs can be built locally. I'd suggest something simpler and more direct like "To build the docs locally during development, see the README file.". What do you think?